### ISSUE TITLE
Scheduled mining pause

### ISSUE THREAD

**Issue by jututt** [xmrig/xmrig#2696]
I'd like this feature to be implemented: scheduled pause of mining of x minutes every xxx minutes,
i.e. adding `"scheduled-pause": 5, 120,` in `config.json` would pause mining for 5 minutes every two hours of working.

Thus to give a break to cpu, fans, power supplies

**Comment by SChernykh** [comment]
It's a bad idea. Constant load is better for hardware longevity. Thermal cycles when it switches between full load and idle are bad.

**Comment by SEVENID** [comment]
Good idea: this way you can mine at night or when your electricity is cheaper without using cron or systemd units.
